Tests/AuthenticationInterceptorTests.swift

```diff
@@ -706,4 +706,101 @@ final class AuthenticationInterceptorTestCase: BaseTestCase {
 
         XCTAssertEqual(request.retryCount, 2)
     }
+
+    // MARK: - Tests - Duplicate Refresh Prevention
+
+    @MainActor
+    func testThatInterceptorPreventsDuplicateRefreshCalls() {
```

Comment:
Without your changes, this test fails because the `refreshCount` is `0`, as you don't actually setup a new credential to be used for refresh.